griptape/drivers/file_manager/griptape_cloud_file_manager_driver.py

```diff
@@ -0,0 +1,160 @@
+from __future__ import annotations
+
+import logging
+import os
+import uuid
+from typing import Optional
+from urllib.parse import urljoin
+
+import requests
+from attrs import Attribute, Factory, define, field
+from azure.core.exceptions import ResourceNotFoundError
+from azure.storage.blob import BlobClient
```

Comment:
Can we get away without using BlobClient? Is it a lot of boilerplate? If not, this needs to be imported using `import_optional_dependency`.

Reply:
We need blob_client (unless we want to avoid Azure SDK). Will make optional